benefit/data/sheets/CaseEvent/CaseEvent.json

```diff
@@ -2495,6 +2495,36 @@
     "PostConditionState": "appealCreated",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2018",
```

Comment:
minor/optional: should we not change these to the current date as per @MohammedHMCTS suggestion?